tests/execution/apollo-federation-entities.md

```diff
@@ -0,0 +1,68 @@
+# Apollo federation \_entities query
```

Comment:
tests are failing for the jit implementation, but t's not because of the implementation and for the following reasons:
- `__typename` doesn't work in jit. Should be solver by pr for typenames
- selection set for upstream graphql doesn't work as expected. Should be resolved as part of fixes for conformance tests